valhalla/baldr/json.h

```diff
@@ -23,20 +23,35 @@ class Jmap;
 using MapPtr = std::shared_ptr<Jmap>;
 class Jarray;
 using ArrayPtr = std::shared_ptr<Jarray>;
-struct fp_t {
+
+// Fixed precision ser
```

Comment:
Nit: Did you mean to truncate this?

Reply:
No, I'm sure what happened here is that a kid threw a toy at my head partway through typing or something.  I don't really remember, but I do have a small lump near my temple....